mlflow/server/js/src/experiment-tracking/components/artifact-view-components/ShowArtifactLoggedModelView.js

```diff
@@ -221,6 +222,15 @@ class ShowArtifactLoggedModelView extends Component {
           />
         </div>
       );
+    } else if (!this.state.hasPyfuncFlavor) {
+      return (
+        <div className='artifact-logged-model-view-no-pyfunc-flavor'>
+          <FormattedMessage
+            defaultMessage="This model doesn't have the pyfunc flavor"
+            description="Error state text when the model doesn't have the pyfunc flavor"
+          />
+        </div>
```

Comment:
It would still be nice to tell users what they can do with the model. E.g. they can reload it via `mlflow.sklearn.load_model()`, perhaps wrap it in a custom pyfunc, etc. I'd imagine Product would have ideas here.

Reply:
Where are these code snippets stored? Can we generate these snippets as part of the log model process instead of statically at rendering time? Would it help to make it flavor specific if dynamically generated at the log model stage?

Reply:
We'd need to write the code snippets. I wouldn't necessarily want to over-index on writing an example that e.g. does inference for every model type. Just discussing the API(s)) available to load the model based on flavor and a link to custom pyfunc model creation would likely be helpful.

Reply:
Thanks for the feedback!

> It would still be nice to tell users what they can do with the model. E.g. they can reload it via mlflow.sklearn.load_model()

This sounds good to me. The idea is to show code like below, right?

````
How to load the model

```
import mlflow
logged_model = "..."

loaded_model = mlflow.sklearn.load_model(logged_model)
```
````

Reply:
@dbczumar 

> a link to custom pyfunc model creation would likely be helpful.


Do you mean this page?

https://www.mlflow.org/docs/latest/python_api/mlflow.pyfunc.html#creating-custom-pyfunc-models